### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by thealphadollar** [tsg-iitkgp/Gymkhana-Forms#14]
@rakaar @mukul-mehta You know what to do here :P

**Comment by rakaar** [comment]
@thealphadollar if there is anything else other than DB part, let me know, I will add.
Also change the name of the repo naa, we don't use for gymkhana forms anymore

**Comment by thealphadollar** [comment]
Yes, we need to roughly add about how this works, what are the different endpoints and how this is used.

@mukul will write about database and deployment. Especially how we login etc.
